frontend-ssr/redux/exercise-submissions/exercise-submissions.reducer.ts

```diff
@@ -40,6 +47,32 @@ export const submissionReducer = (state = initialState, action: { type: string;
           return submission;
         }),
       };
+    case EXERCISE_SUBMISSIONS.LOAD: {
+      const { n } = action.payload;
+      const submissions = getMockSubmissions(n);
+      return {
+        ...state,
+        submissions,
```

Comment:
Here we should append the new submissions to the list, not replace them altogether. 

Also, please increae the page numbe as well.